### DISCUSSION TITLE
v1.0 Update of info in Discourse & GitHub

### DISCUSSION THREAD

**Discussion by emyoulation** [gramps-project/gramps-web-api#529] [General]
The interconnections of the associated pieces are somewhat complex.  Could you update the versioning information (or README.md files) in the various  3 Repositories, 4 Discourse threads, and 2 DigitalOcean listings to cross-reference the pieces and point to the Release pages? (So you don't have to keep manually updating beyond the 1.0 release announcement.) 

_Links have been included to aid in composing a re-usable section._

<h2>Interconnected Repositories on GitHub</h2>

<h3>Gramps Web _(formerly?) gramps-webapi_</h3>

* https://github.com/gramps-project/gramps-webapi
* https://github.com/gramps-project/gramps-webapi/releases
version 1.0.0
* https://github.com/gramps-project/gramps-webapi/pull/383/commits/a1feb305d6fe62bfb234805906adcc82e54949b1
meta version 0.7.0 bumped to 1.0.0 on 24 Jun 2023
* https://github.com/gramps-project/gramps-webapi/releases/tag/v1.0.0

<h3>Gramps.js  </h3>

* https://github.com/gramps-project/Gramps.js 

* https://github.com/gramps-project/Gramps.js/releases  <br />version 0.21

<h3>Gramps Web Sync add-on (Gramps plug-in) tool</h3>

gramps-addon-webapisync

* https://github.com/DavidMStraub/gramps-addon-webapisync
* https://github.com/DavidMStraub/gramps-addon-webapisync/releases <br/ >version ???? (no releases published; version="1.0" in the Gramps Plugin Registration file.)

> Installation
> This addon requires Gramps 5.1 running on Python 3.6 or newer, as well as a Gramps Web API instance running version 0.3.0 or newer.

<h2>Mentions on Discourse:</h2>

* https://gramps.discourse.group/t/gramps-web-digitalocean-1-click-app/2733
* <strike>https://gramps.discourse.group/t/gramps-web-api-gramps-js-progress/1925</strike>
* <strike>https://gramps.discourse.group/t/gramps-js-a-web-frontend-to-your-gramps-database/1626</strike>
* <strike>https://gramps.discourse.group/t/gramps-web-api-first-release/1609</strike>

<h2>DigitalOcean</h2>
 
* https://docs.digitalocean.com/products/marketplace/catalog/gramps-web/
> Package 	Version 	License
> Gramps Web 	0.13.3+ 	AGPLv3
* https://marketplace.digitalocean.com/apps/gramps-web
> Gramps Web
> by The Gramps Project
> Version 1.0   OS Ubuntu 20.04
> Package     Version  License
> Gramps Web  0.13.3+  AGPLv3

**Comment by emyoulation** [comment]
We can follow your lead when updating the wiki sections:
* https://gramps-project.org/wiki/index.php/Web_Solutions_for_Gramps
* https://gramps-project.org/wiki/index.php/Addon:Gramps_Web
* Gramps Web Sync is not yet listed in the Third-Party Addons nor does it have its own page

**Comment by DavidMStraub** [comment]
There are still two separate version numbers, and I think there is ample reason to keep them separate.

The updated doc site has a comment about it: https://www.grampsweb.org/dev/

Conncerning users, we need to make it visible to them which versions they are actually using, new issue: 
https://github.com/gramps-project/Gramps.js/issues/233

**Comment by emyoulation** [comment]
Ok.  Separate version number for Gramps Web, Gramps.js and Gramps Web Sync (webapisync?) make sense.  

But why are there 2 versions for Gramps Web in the DigitalOcean listing?   version 1.0.0 and  0.13.3+ ?   

Version [v0.1](https://github.com/gramps-project/gramps-webapi/releases/tag/v0.1) was 3 Jul 2021.  Is 0.13.3 the first version they supported?  And is there a forced update for older droplets someone might have subscribed to?  Having versions prior to 1.0.0 in the wild on their servers might be a bad thing.

**Comment by DavidMStraub** [comment]
> But why are there 2 versions for Gramps Web in the DigitalOcean listing? version 1.0.0 and 0.13.3+ ?

1.0.0 is the first release of the DO 1-click app itself.

On first release, it forced me to enter a version number of Gramps Web, but it's silly since the newest version is pulled upon installing it. So I put "0.13.3+" with the + indicated that it will be some higher version...
